library/core/functions.general.php

```diff
@@ -2109,35 +2109,29 @@ function jsonEncodeChecked($value, $options = null) {
         $advanced = (PHP_VERSION_ID >= 50500);
 
         if ($options === null) {
-            if ($advanced) {
-                $options = JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES | JSON_PARTIAL_OUTPUT_ON_ERROR;
-            } else {
-                $options = JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES;
-            }
+            $options = JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES;
```

Comment:
Having partial response is not necessarily good. Having null values instead of the correct data can cause potential problems down the line.
It is also harder to debug!

Having `JSON_PARTIAL_OUTPUT_ON_ERROR` also prevent some errors to be detected because they will never happen. ie JSON_ERROR_UNSUPPORTED_TYPE